test/unit/parsed-statement.test.js

```diff
@@ -13,6 +13,7 @@ const { match } = require('../lib/custom-assertions')
 
 const Transaction = require('../../lib/transaction')
```

Comment:
these tests are asserting the new `database` metrics recorder. parsed statement is nothing more than a class now. I would move these tests to `test/unit/metrics-recroders/

Reply:
Got it. I'll move them.